src/mix/api/applications-types.ts

```diff
@@ -9,11 +9,13 @@
 import {Expand} from './shared-types'
 
 /**
- * Application credentials view to return.
- * - ACV_VIEW_UNSPECIFIED: Returns credentials details without including clients
- * - ACV_FULL: Returns all credentials details, including list of clients
+ * Application view to return.
+ * - AV_VIEW_UNSPECIFIED: Returns application details without including application configurations
```

Comment:
Fix also the `ACV_FULL` and `ACV_VIEW_UNSPECIFIED` found a few lines below as well.